test/e2e/e2e_suite_test.go

```diff
@@ -493,11 +494,11 @@ func getLatestSkuForMinor(version string, skus semver.Versions) string {
 				break
 			}
 		}
-	} else {
-		v, err := semver.ParseTolerant(version)
-		Expect(err).NotTo(HaveOccurred())
+	} else if v, err := semver.ParseTolerant(version); err == nil {
```

Comment:
parse tolerant allows for versions with or without a v in it.  I think this is ok here but just a note

https://pkg.go.dev/github.com/blang/semver#ParseTolerant

Reply:
Yes, this is the only way to parse a version with a "v" prefix (which we use in k8s versions). `Parse` won't allow for a "v" prefix.